lib/test_lib.py

```diff
@@ -571,7 +571,7 @@ def SetupClients(self, nr_clients):
         fd.Flush()
 
         index.AddClient(fd)
-
+    index.Close()
```

Comment:
I went ahead and switched SetupClients to use a with statement.  This seems like the more correct behavior for SetupClients either way.